common/protob/messages-ethereum.proto

```diff
@@ -66,7 +66,26 @@ message EthereumSignTx {
     optional bytes data_initial_chunk = 7;  // The initial data chunk (<= 1024 bytes)
     optional uint32 data_length = 8;        // Length of transaction payload
     optional uint32 chain_id = 9;           // Chain Id for EIP 155
-    optional uint32 tx_type = 10;           // (only for Wanchain)
+    optional uint32 tx_type = 10;           // Used for Wanchain
+}
+
+message EthereumSignTxEIP1559 {
+    repeated uint32 address_n = 1;          // BIP-32 path to derive the key from master node
+    required bytes nonce = 2;               // <=256 bit unsigned big endian
+    required bytes max_gas_fee = 3;        // <=256 bit unsigned big endian (in wei)
+    required bytes max_priority_fee = 4;   // <=256 bit unsigned big endian (in wei)
+    required bytes gas_limit = 5;           // <=256 bit unsigned big endian
+    optional string to = 11;                // recipient address
```

Comment:
also please renumber the fields as they should be :)

(the reason EthereumSignTx is not numbered sequentially is because of an incompatible change in the past)

Reply:
Done